fiftyone/core/odm/database.py

```
@@ -836,11 +836,11 @@ def drop_orphan_stores(dry_run=False):
             _delete_stores(conn, orphan_store_ids)
 
 
-def drop_orphan_temporal_tags(dry_run=False):
-    """Drops all orphan multimodal temporal tags from the database.
+def drop_orphan_tags(dry_run=False):
```

Ruff (0.15.17)

[warning] Boolean default positional argument in function definition

(FBT002)